docs/docs/components/guidelines/README.md

```diff
@@ -391,7 +391,7 @@ connect their account and enter a search term. The remaining fields are optional
 for users who want to filter the results, but they do not require any action to
 activate the source:
 
-![image-20210326151930885](images/image-20210326151930885.png)
+![image-20210326151930885](../../../../images/image-20210326151930885.png)
```

Comment:
You'll also need to move these images into the local `images/` subdirectory (see main comment on why the Vercel build failed)